src/core/error_handling.c

```diff
@@ -219,24 +219,13 @@ wrap_exception()
   return jserror;
 }
 
-EM_JS(void, log_python_error, (JsRef jserror), {
-  // If a js error occurs in here, it's a weird edge case. This will probably
-  // never happen, but for maximum paranoia let's double check.
-  try {
-    let msg = Hiwire.get_value(jserror).message;
```

Comment:
This function doesn't seem to do anything. Have you tested that this change actually does what you want?